pkg/datagatherer/k8s/dynamic.go

```diff
@@ -211,12 +213,25 @@ func redactList(list *unstructured.UnstructuredList) error {
 						}
 						metadata["annotations"] = annotations
 					}
-					secret.Object["metadata"] = metadata
+
+					// update the metadata after an modifications
+					object.Object["metadata"] = metadata
 				}
 				// break when the object has been processed as a secret, no
 				// other kinds have redact modifications
 				break
 			}
+
+			metadata, metadataPresent := object.Object["metadata"].(map[string]interface{})
+			if metadataPresent {
+				// Drop managed fields if set
+				if _, present := metadata["managedFields"]; present {
+					delete(metadata, "managedFields")
+				}
+			}
+
+			// update the metadata after an modifications
+			object.Object["metadata"] = metadata
```

Comment:
Is this needed?

Reply:
Huh, we don't. I just copied it from above. Removed in 2ba21c0092016487bce2a1bdba2525e60291cd5b